glossary/en/terms/resource/index.md

```diff
@@ -5,4 +5,4 @@ title: Resource
 ---
 
 
-CKAN uses this term to denote one of the individual data objects (a file such as a spreadsheet, or an [API](/glossary/en/terms/api/)) in a [dataset](/glossary/en/terms/dataset/).
+CKAN uses this term to denote one of the individual data objects (a file such as a spreadsheet, or an [API](../api/)) in a [dataset](../dataset/).
```

Comment:
@morchickit Here again, the second `)` has its matching opening parenthesis before `a file such`.